workbench-core/base/src/interfaces/FilterRequest.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { QueryParameterFilter } from './queryParameterFilter';
+
+export interface FilterRequest {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  [key: string]: QueryParameterFilter<any> | undefined;
```

Comment:
could `unknown` be used here instead of `any`?

Reply:
changed to unknown